src/app/nav.tsx

```diff
@@ -0,0 +1,111 @@
+"use client";
+import styles from "./nav.module.css";
+import React from "react";
+import { useState } from "react";
+
+function Navbar() {
+  const navItems = ["Pokemon Lab", "Battle", "Poke Mart", "My Pokedex"];
+  const [isNavExpanded, setIsNavExpanded] = useState(false);
+  return (
+    <>
+      <div className={styles.navContainer}>
+        <div className={styles.icons}>
+          <img
+            src="/images/hamburger.png"
+            className={styles.hamburgerIcon}
+            onClick={() => {
```

Comment:
It is not good practice to set onClick like this on images. You should use actual clickable elements like `<button>` or `<a>`.